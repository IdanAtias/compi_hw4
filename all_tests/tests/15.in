bool a;
bool b;
bool c = a or b;

