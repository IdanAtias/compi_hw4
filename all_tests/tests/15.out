-- end scope --
var:a:0:BOOL
var:b:1:BOOL
var:c:2:BOOL
